src/HearThis/Script/MultiVoiceChapter.cs

```diff
@@ -45,7 +42,18 @@ public MultiVoiceChapter(XElement chapter, MultiVoiceScriptProvider provider)
 
 		private static MultiVoiceBlock[] GetBlocks(MultiVoiceScriptProvider provider, XElement block, ref int index)
 		{
-			var text = block.Element("text")?.Value.Trim() ?? "";
+			string text;
+			switch (provider?.Version?.Major)
+			{
+				// If greater than max allowable in this version, we already handled that in MultiVoiceScriptProvider constructor
+				case 2:
+					text = String.Join("", block?.Element("vernacularText")?.Elements("text").Select(textElement => textElement?.Value ?? "") ?? new string[0]);
+					break;
+				default:
+					text = block.Element("text")?.Value.Trim() ?? "";
```

Comment:
Looking at the calling code, it does not appear that block could ever be null. If I'm missing something and it can be, then both cases should handle it.